modules/representation_management/spec/support/rswag_config.rb

```diff
@@ -160,6 +162,76 @@ def power_of_attorney_detailed_attributes
     }
   end
 
+  def veteran_service_representative_schema
```

Comment:
There's still snake case here

Reply:
This is a ruby method name, not a key that will show up in the swagger doc.  It is legal in ruby to use camel case in method names like
```ruby
def thisIsTheBestMethod
end
```
but it is very unconventional.  I searched the `vets-api` repo and out of 15K+ methods defined only one has a capital letter in it.

I would leave this as is.